### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pherterich**
### Summary

Feeding back the info from #430 for completeness.

### List of changes proposed in this PR (pull-request)

* adding more info on talks and all the links I'm aware off

### What should a reviewer concentrate their feedback on?

- [ ] typos, formatting mess etc.
- [ ] Are there any further talks or presentations missing?

### Acknowledging contributors

By submitting this pull request I confirm that:

- [x] all contributors to this pull request who wish to be included are named in [contributors.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/contributors.md).

### CODE REVIEW

**Comment by rainsworth** [comment]
I get 404 pages for the links to the pdf slides, it doesn't look like they've been added?

File: conferences/README.md [hunk 1]
```diff
@@ -16,4 +16,10 @@ THANK YOU to every member of the community who spreads the word! :sparkling_hear
...
 | Talk at MRC-BSU | 20 March 2019 | | [Whitaker_MRCBSU.pdf](presentations/MRCBSU_20190320/Whitaker_MRCBSU_TheTuringWay_March2019.pdf)<br>(File name shortened for better readability) | [10.5281/zenodo.2599904](https://doi.org/10.5281/zenodo.2599904)
 | [Tools, Practices and Systems Scoping Workshop](https://www.turing.ac.uk/events/tools-practices-and-systems-data-science-and-artificial-intelligence-scoping-workshop) | 21 March 2019 | | [TuringWayPoster_TPSScopingWorkshop.pdf](presentations/TPS_ScopingWorkshop_20190321/Whitaker_TuringWayPoster_TPSScopingWorkshop.pdf) | [10.5281/zenodo.2598546](https://doi.org/10.5281/zenodo.2598546) |
+| [SSI Collaborations Workshop](https://www.software.ac.uk/cw19) | 1-3 April 2019 | [CollabW demo](https://software.ac.uk/cw19/mini-workshops-and-demos-sessions#session-3) | [CollabW slides](https://github.com/alan-turing-institute/the-turing-way/blob/f5b835024669da97c5c5c5c57491e3b227342165/project_management/conferences/presentations/CollabW19/Whitaker_CollabW19_April2019.pdf) | [10.5281/zenodo.2621279](https://doi.org/10.5281/zenodo.2621279) |
```
Comment (rainsworth) [inline conferences/README.md]: Changing
```[CollabW slides](https://github.com/alan-turing-institute/the-turing-way/blob/f5b835024669da97c5c5c5c57491e3b227342165/project_management/conferences/presentations/CollabW19/Whitaker_CollabW19_April2019.pdf)```
to
```[CollabW slides](presentations/CollabW19/Whitaker_CollabW19_April2019.pdf)```
should fix this link.

**Comment by pherterich** [comment]
Ahh, have copied them from #430 and they are broken. Will need to fix it in both places...

**Comment by pherterich** [comment]
Not entirely sure what my logic was but I don't think it worked. Happy for this to be closed and we do a proper update.

**Comment by malvikasharan** [comment]
@KirstieJane please approve this to be closed. As per the discussion, this PR is not gonna be merged.

**Pull request closed without merging**